sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -137,20 +130,18 @@
     "frequency/@units": "@attrs:metadata/scientificMetadata/Source/repetition_rate/unit",
     "associated_beam": "/entry/instrument/beam_probe"
   },
-  "/ENTRY[entry]/INSTRUMENT[instrument]/beam_TYPE[beam_probe]": {
+  "/ENTRY[entry]/INSTRUMENT[instrument]/beamTYPE[beam_probe]": {
     "distance": 0.0,
     "distance/@units": "mm",
     "incident_energy": "@attrs:metadata/scientificMetadata/Source/photon_energy/value",
-    "incident_energy/@units": "@attrs:metadata/scientificMetadata/Source/dispersion/unit",
-    "incident_energy_dispersion": "@attrs:metadata/scientificMetadata/Source/dispersion/value",
-    "incident_energy_dispersion/@units": "@attrs:metadata/scientificMetadata/Source/dispersion/unit",
```

Comment:
I am not so sure what this actually means. Is this a property of the grating? There is a base class NXgrating for grating monochromators, maybe this can be used here.

Reply:
Yes, this is coming directly as beamline/grating metadata which we are reading in eV/mm - providing the bandwidth for the corresponding exit slit width.

Reply:
This one I don't know right now, but I will discuss where this can go or if we need a new field.

Reply:
Do you know what the propper name for this is, and e.g. if it is related to any concept in an ontology?

Reply:
We have added a respective concept here: https://github.com/FAIRmat-NFDI/nexus_definitions/pull/257
Still need to find its way into the next release of pynxtools

Reply:
done now.